.circleci/config.yml

```diff
@@ -30,7 +30,7 @@ aliases:
         command: |
           git clone https://github.com/kubernetes-sigs/kind.git /root/src/sigs.k8s.io/kind
```

Comment:
Do we still need to build our own kind?

Reply:
I don't think so, but planned to change this within a distinct PR.

Reply:
Ok then!